assets/js/admin/course-theme-sidebar.js

```diff
@@ -3,49 +3,152 @@
+		const blocks = getBlocks();
+		const courseOutline = getFirstBlockByName(
+			courseOutlineBlockName,
+			blocks
+		);
+
+		if ( ! courseOutline ) {
+			return {};
+		}
+
+		for ( const moduleOrLesson of getBlocks( courseOutline.clientId ) ) {
+			if ( canPreview( moduleOrLesson ) ) {
+				return getBlockAttributes( moduleOrLesson.clientId );
+			}
+
+			if ( moduleOrLesson.name === courseModuleBlockName ) {
+				for ( const lesson of getBlocks( moduleOrLesson.clientId ) ) {
+					if ( canPreview( lesson ) ) {
+						return getBlockAttributes( lesson.clientId );
+					}
+				}
+			}
+		}
+
+		return {};
+	} );
+
+	let previewUrl = '';
+	if ( firstLesson?.id && currentPost?.id ) {
+		if (
+			globalLearningModeEnabled ||
+			currentPost.meta._course_theme === SENSEI_THEME
+		) {
+			previewUrl = `/?p=${ firstLesson.id }`;
+		} else {
+			previewUrl = `/?p=${ firstLesson.id }&learn=1&${ SENSEI_PREVIEW_QUERY }=${ currentPost.id }`;
+		}
+
+		if ( firstLesson.draft ) {
+			previewUrl = `${ previewUrl }&post_type=lesson`;
+		}
+	}
+
+	let customizerUrl = '';
+	if ( previewUrl ) {
+		customizerUrl = `/wp-admin/customize.php?url=${ encodeURIComponent(
+			previewUrl
+		) }`;
+	}
+
+	return { previewUrl, customizerUrl };
+};
 
 /**
  * Course Theme Sidebar component.
  */
 const CourseThemeSidebar = () => {
+	const globalLearningModeEnabled =
+		window.sensei?.senseiSettings?.sensei_learning_mode_all || false;
 	const [ theme, setTheme ] = useCourseMeta( '_course_theme' );
+	const { previewUrl, customizerUrl } = usePreviewAndCustomizerLinks();
 
 	return (
 		<PluginDocumentSettingPanel
 			name="sensei-course-theme"
-			title={ __( 'Course Theme', 'sensei-lms' ) }
+			title={ __( 'Course Styles', 'sensei-lms' ) }
 		>
-			<p>
-				{ __(
-					'This does not change the theme of your site. It only applies to logged in users when viewing the course.',
-					'sensei-lms'
-				) }
-			</p>
-			<SelectControl
-				label={ __( 'Theme', 'sensei-lms' ) }
-				hideLabelFromVision
-				value={ theme }
-				options={ [
-					{
-						label: __( 'Sensei Theme', 'sensei-lms' ),
-						value: SENSEI_THEME,
-					},
-					{
-						label: __( 'WordPress Theme', 'sensei-lms' ),
-						value: WORDPRESS_THEME,
-					},
-				] }
-				onChange={ setTheme }
-			/>
+			{ globalLearningModeEnabled ? (
+				<p className="sensei-course-theme-toggle__global-overwrite-notice">
+					<a href="/wp-admin/admin.php?page=sensei-settings#course-settings">
+						{ __(
+							'Learning Mode is enabled globally.',
+							'sensei-lms'
+						) }
+					</a>
+				</p>
+			) : (
+				<ToggleControl
+					label={ __( 'Learning mode', 'sensei-lms' ) }
+					help={
+						<>
+							<p className="sensei-course-theme-toggle__description">
+								{ __(
+									'Enable this mode to show an immersive and dedicated view for the course, lessons, and quizzes.',
+									'sensei-lms'
+								) }
+							</p>
+							{ previewUrl && (
+								<p className="sensei-course-theme-toggle__preview">
```

Comment:
Is there some reason for the preview to be part of the help text and the customizer to be out of this? If not, I think we could add both links after the toggle. It could also be a `ul` to be more semantic. WDYT?

Reply:
No reason anymore. It was part of the field helper text before so it would properly align with the toggle field label as in the designs (see below). But now, since we are not sticking to designs strictly, I took it out of the `help` prop of `ToggleControl` component.

I put them in the `p` tags instead of `ul>li` because the latter needs additional CSS to align them properly. So it will require some additional CSS file to be created and played with margins and paddings to keep em in `ul>li` tags. But if you don't like the `p` tag, then let me know and I will update them to be in `ul>li` tags. 👍 

#### Original Design
<img width="351" alt="Screen Shot 2022-01-11 at 12 54 48 PM" src="https://user-images.githubusercontent.com/2578542/148911261-3192b49d-f643-46e5-a7a3-356c22718ebf.png">
 
#### Current Implementation
<img width="271" alt="Screen Shot 2022-01-11 at 12 59 16 PM" src="https://user-images.githubusercontent.com/2578542/148912169-72da2395-720b-4c37-ba04-c7ca638f11d5.png">